README.md

```diff
@@ -8,14 +8,14 @@ Usage:
 
 1. `OVERCAST_USERNAME=you@there.com OVERCAST_PASSWORD=1234 python overcast-sonos.py`
 
-1. Go to `http://<SONOS_IP>:1400/customsd.htm`, and enter:
+1. Go to `http://<SONOS_IP>:8140/customsd.htm`, and enter:
     
     - SID - some unique SID (255 works if you've not done this before
     - Service Name - some name, `overcast` works
-    - Endpoint URL and Secure Endpoint URL: `http://<YOUR_IP>:8080/overcast-sonos`
+    - Endpoint URL and Secure Endpoint URL: `http://<YOUR_IP>:8140/overcast-sonos`
```

Comment:
Another port question here - is this just for consistency with above, or something else?